home/tests/test_content_import_export.py

```diff
@@ -1329,7 +1329,7 @@ def test_invalid_page_already_in_db(self, csv_impexp: ImportExport) -> None:
         assert e.value.row_num == 3
         # FIXME: Find a better way to represent this.
         assert e.value.message == [
-            "Validation error: example_values - The number of example values provided (1) does not match the number of variables used in the template (3)"
+            "Validation error at row 3: example_values - The number of example values provided (1) does not match the number of variables used in the template (3)"
```

Comment:
Do we need the row number added here? The row number is already included in the Exception, and the code that converts the Exception to an error message for the user adds the row number to the message, so now we'll have duplicate row numbers in the error message.